discovery-provider/src/queries/query_helpers.py

```diff
@@ -102,132 +103,44 @@ def populate_user_metadata(session, user_ids, users, current_user_id, with_track
     if current_user_id:
         enqueue_balance_refresh(redis, [current_user_id])
 
-    # build dict of user id --> track count
-    track_counts = (
+    aggregate_user = (
```

Comment:
Looks like we don't use the with_track_save_count param anymore, correct?

Reply:
We do call it in the `get_users_account.py` file.
https://github.com/AudiusProject/audius-protocol/blob/0db6834e6c8baf4764e4d7561172f435fbd47d27/discovery-provider/src/queries/get_users_account.py#L40-L41